livekit-plugins/livekit-plugins-aws/livekit/plugins/aws/experimental/realtime/realtime_model.py

```diff
@@ -231,25 +270,39 @@ class RealtimeModel(llm.RealtimeModel):
     spawns a RealtimeSession when session() is invoked.
     """
 
+    class SupportedModels:
+        """Supported Nova Sonic realtime models."""
+
+        NOVA_SONIC_1 = "amazon.nova-sonic-v1:0"
+        NOVA_SONIC_2 = "amazon.nova-2-sonic-v1:0"
+
     def __init__(
         self,
         *,
-        voice: NotGivenOr[VOICE_ID] = NOT_GIVEN,
+        model: str = SupportedModels.NOVA_SONIC_1,
+        modalities: MODALITIES = "audio",
+        voice: NotGivenOr[str] = NOT_GIVEN,
```

Comment:
I think it would be beneficial to utilize the voice Literals and modify the docstring so folks can be pointed to the supported voices list. Also would need to be imported at the top
```suggestion
        voice: NotGivenOr[SONIC1_VOICES | SONIC2_VOICES | str] = NOT_GIVEN,
```

Reply:
cef1ffef0d4feb678b58357a978808cf92acedb8